tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -138,6 +150,43 @@ def loadbalancer_setup(
     }
 
 
+@pytest.fixture(scope='module')
+def enable_subscriptions(module_target_sat, module_entitlement_manifest_org, version):
+    """Enable and sync RHEL rpms repos"""
```

Comment:
there is a fixture `content_for_client` already that syncs custom RH repos, why do we need to sync it again from CDN? Do you think it's redundant or is it intended?